{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "973c8d50",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import json\n",
    "\n",
    "import time\n",
    "from Bio import Entrez, Medline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "03a6e53f",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extrair_info(search_term, email):\n",
    "    Entrez.email = email\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=search_term )\n",
    "    record=Entrez.read(handle)\n",
    "    handle.close()\n",
    "    id_list=record[\"IdList\"]\n",
    "    handle=Entrez.efetch(db=\"pubmed\", id=id_list, rettype=\"medline\", retmode=\"text\")\n",
    "    records=Medline.parse(handle)\n",
    "    results=[]\n",
    "    for record in records:\n",
    "        result={\"title\": record.get(\"TI\", \"\"), \"author\": record.get(\"AU\", \"\"), \"abstract\": record.get(\"AB\", \"\")}\n",
    "        results.append(result)\n",
    "    handle.close()\n",
    "    return (results)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "83f42f98",
   "metadata": {},
   "outputs": [],
   "source": [
    "def gravarBD(bd,fnome):\n",
    "    f=open(fnome, \"w\", encoding='utf-8')\n",
    "    json.dump(bd,f)\n",
    "    return bd"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bbb1dc5c",
   "metadata": {},
   "source": [
    "# Para o PNPLA3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "07b0b74d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'title': 'Association of HSD17B13 and PNPLA3 with liver enzymes and fibrosis in Hispanic/Latino individuals of diverse genetic ancestries.',\n",
       "  'author': ['Rutledge SM',\n",
       "   'Soper ER',\n",
       "   'Ma N',\n",
       "   'Pejaver V',\n",
       "   'Friedman S',\n",
       "   'Branch AD',\n",
       "   'Kenny EE',\n",
       "   'Belbin GM',\n",
       "   'Abul-Husn NS'],\n",
       "  'abstract': 'BACKGROUND/AIMS: Genetic variants affecting liver disease risk vary among racial and ethnic groups. Hispanics/Latinos in the U.S. have a high prevalence of PNPLA3 I148M, which increases liver disease risk, and a low prevalence of HSD17B13 predicted loss-of-function (pLoF) variants, which reduce risk. Less is known about the prevalence of liver disease-associated variants among Hispanic/Latino subpopulations defined by country-of-origin and genetic ancestry. We evaluated the prevalence of HSD17B13 pLoF variants and PNPLA3 I148M, and their associations with quantitative liver phenotypes in Hispanic/Latino participants of an electronic health record-linked biobank in New York City. METHODS: This study included 8,739 adult Hispanic/Latino participants of the BioMe biobank with genotyping and exome sequencing data. We estimated the prevalence of Hispanic/Latino individuals harboring HSD17B13 and PNPLA3 variants, stratified by genetic ancestry, and performed association analyses between variants and liver enzymes and Fibrosis-4 (FIB-4) scores. RESULTS: Individuals with ancestry from Ecuador and Mexico had the lowest frequency of HSD17B13 pLoF variants (10%/7%) and the highest frequency of PNPLA3 I148M (54%/65%). These ancestry groups had the highest outpatient alanine aminotransferase (ALT) and aspartate aminotransferase (AST) levels, and largest proportion of individuals with a FIB-4 score >2.67. HSD17B13 pLoF variants were associated with reduced ALT (p = 0.002), AST (p = <0.001), and FIB-4 score (p = 0.045). PNPLA3 I148M was associated with increased ALT, AST, and FIB-4 score (p<0.001 for all). HSD17B13 pLoF variants mitigated the increase in ALT conferred by PNPLA3 I148M (p = 0.006). CONCLUSIONS: Variation in HSD17B13 and PNPLA3 variants across genetic ancestry groups may contribute to differential risk for liver fibrosis among Hispanic/Latino individuals.'},\n",
       " {'title': 'COVID-19 and NAFLD: biological insights from multi-omics data.',\n",
       "  'author': ['Pirola CJ', 'Sookoian S'],\n",
       "  'abstract': 'We explored the shared pathophysiological mechanisms between COVID-19 and non-alcoholic fatty liver disease (NAFLD) by integrating multi-omics data. We studied common genetic risk factors and underlying biological processes using functional enrichment analysis. To understand the sex-specific pathways involved in the clinical course of SARS-CoV-2 infection, we processed sex-stratified data from COVID-19 genome-wide association datasets. We further explored the transcriptional signature of the liver cells in healthy and COVID-19 tissue specimens. We also integrated genetic and metabolomic information. We found that COVID-19 and NAFLD share biological disease mechanisms, including pathways that regulate the inflammatory and lipopolysaccharide response. Single-cell transcriptomics revealed enrichment of complement-related pathways in Kupffer cells, syndecan-mediated signaling in plasma cells, and epithelial-to-mesenchymal transition in hepatic stellate cells. The strategy of pathway-level analysis of genomic and metabolomic data uncovered L-lactic acid, Krebs cycle intermediate compounds, arachidonic acid, and cortisol among the most prominent shared metabolites.'},\n",
       " {'title': 'Association of Inherited Genetic Factors With Drug-Induced Hepatic Damage Among Children With Acute Lymphoblastic Leukemia.',\n",
       "  'author': ['Yang W',\n",
       "   'Karol SE',\n",
       "   'Hoshitsuki K',\n",
       "   'Lee S',\n",
       "   'Larsen EC',\n",
       "   'Winick N',\n",
       "   'Carroll WL',\n",
       "   'Loh ML',\n",
       "   'Raetz EA',\n",
       "   'Hunger SP',\n",
       "   'Winter SS',\n",
       "   'Dunsmore KP',\n",
       "   'Devidas M',\n",
       "   'Relling MV',\n",
       "   'Yang JJ'],\n",
       "  'abstract': \"IMPORTANCE: Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. Hepatotoxic effects, including hyperbilirubinemia and elevated alanine aminotransferase (ALT) and aspartate aminotransferase (AST) levels, are common during all phases of therapy and are linked to several chemotherapeutic agents, including asparaginase, mercaptopurine, and methotrexate. OBJECTIVE: To determine which genetic variants were associated with hyperbilirubinemia and elevated ALT and AST levels in children, adolescents, and young adults treated for ALL. DESIGN, SETTING, AND PARTICIPANTS: This retrospective analysis of a multiethnic genome-wide association study was conducted between January 1, 2019, and April 15, 2022, including patients treated as part of Children's Oncology Group (COG) trials with centers in the United States, Canada, and Australia, which accrued data from December 29, 2003, to January 21, 2011 (AALL0232), and from January 22, 2007, to July 24, 2014 (AALL0434). Germline genotypes were interrogated using genome-wide arrays and imputed using a National Institutes of Health TOPMed Imputation server. Mixed-effects logistic regressions were used to account for multiple episodes for an individual patient. Genotype x treatment phase interaction was tested to uncover phase-specific genetic risk factors. EXPOSURES: Total duration of multiagent protocol chemotherapy ranging from 2.5 to 3.5 years. MAIN OUTCOMES AND MEASURES: The primary outcomes were National Cancer Institute Common Terminology Criteria for Adverse Events (version 4) hyperbilirubinemia of grade 3 or higher and elevated liver ALT and AST levels. RESULTS: A total of 3557 participants were included in the analysis (2179 [61.3%] male; median age, 11.1 [range, 1-30] years). Among 576 known variants associated with these liver function test results in the general population, UGT1A1 variant rs887829 and PNPLA3 variant rs738409 were associated with increased risk of hyperbilirubinemia (odds ratio [OR], 2.18 [95% CI, 1.89-2.53]; P = 6.7 x 10-27) and ALT and AST levels (OR, 1.27 [95% CI, 1.15-1.40]; P = 3.7 x 10-7), respectively, during treatment for ALL. Corresponding polygenic risk scores were associated with hepatotoxic effects across all therapy phases and were largely driven by UGT1A1 and PNPLA3 variants. Genome-wide association analysis revealed an age-specific variant near the CPT1A gene that was only associated with elevated ALT and AST levels among patients younger than 10 years (OR, 1.28 [95% CI, 1.18-1.39]; P = 8.7 x 10-10). CONCLUSIONS AND RELEVANCE: These results suggest a strong genetic basis for interpatient variability in hyperbilirubinemia and aminotransferase level elevations during leukemia chemotherapy.\"},\n",
       " {'title': 'A case of NASH with genetic predisposition successfully treated with an SGLT2 inhibitor: a possible involvement of mitochondrial dysfunction.',\n",
       "  'author': ['Nakajima R',\n",
       "   'Sekiya M',\n",
       "   'Furuta Y',\n",
       "   'Miyamoto T',\n",
       "   'Sato M',\n",
       "   'Fukuda K',\n",
       "   'Hattori K',\n",
       "   'Suehara Y',\n",
       "   'Sakata-Yanagimoto M',\n",
       "   'Chiba S',\n",
       "   'Okajima Y',\n",
       "   'Matsuzaka T',\n",
       "   'Takase S',\n",
       "   'Takanashi M',\n",
       "   'Okazaki H',\n",
       "   'Takashima Y',\n",
       "   'Yuhara M',\n",
       "   'Mitani Y',\n",
       "   'Matsumoto N',\n",
       "   'Murayama Y',\n",
       "   'Ohyama Osawa M',\n",
       "   'Ohuchi N',\n",
       "   'Yamazaki D',\n",
       "   'Mori S',\n",
       "   'Sugano Y',\n",
       "   'Osaki Y',\n",
       "   'Iwasaki H',\n",
       "   'Suzuki H',\n",
       "   'Shimano H'],\n",
       "  'abstract': 'SUMMARY: In this study, we herein describe a 47-year-old Japanese woman who manifested inheritable non-alcoholic steatohepatitis (NASH) and severe dyslipidemia. Interestingly, her NASH progression was ameliorated by treatment with a sodium-glucose co-transporter 2 (SGLT2) inhibitor. This inheritability prompted us to comprehensively decode her genomic information using whole-exome sequencing. We found the well-established I148M mutation in PNPLA3 as well as mutations in LGALS3 and PEMT for her NASH. Mutations in GCKR may contribute to both NASH and dyslipidemia. We further mined gene mutations potentially responsible for her manifestations that led to the identification of a novel M188fs mutation in MUL1 that may be causally associated with her mitochondrial dysfunction. Our case may provide some clues to better understand this spectrum of disease as well as the rationale for selecting medications. LEARNING POINTS: While the PNPLA3 I148M mutation is well-established, accumulation of other mutations may accelerate susceptibility to non-alcoholic steatohepatitis (NASH). NASH and dyslipidemia may be intertwined biochemically and genetically through several key genes. SGLT2 inhibitors emerge as promising treatment for NASH albeit with interindividual variation in efficacy. Genetic background may explain the mechanisms behind the variation. A novel dysfunctional mutation in MUL1 may lead to metabolic inflexibilities through impaired mitochondrial dynamics and function.'},\n",
       " {'title': 'Molecular Screening via Sanger Sequencing of the Genetic Variants in Non-Alcoholic Fatty Liver Disease Subjects in the Saudi Population: A Hospital-Based Study.',\n",
       "  'author': ['Alsaif F',\n",
       "   'Al-Hamoudi W',\n",
       "   'Alotaiby M',\n",
       "   'Alsadoon A',\n",
       "   'Almayouf M',\n",
       "   'Almadany H',\n",
       "   'Abuhaimed J',\n",
       "   'Ghufran N',\n",
       "   'Merajuddin A',\n",
       "   'Ali Khan I'],\n",
       "  'abstract': 'Non-alcoholic fatty liver disease (NAFLD) is one of the most common liver diseases, along with steatosis and non-alcoholic steatohepatitis (NASH), and is associated with cirrhosis and hepatocellular carcinoma. Candidate gene and genome-wide association studies have validated the relationships between NAFLD, NASH, PNPLA3, TM6SF2, and HFE. The present study utilized five polymorphisms in three genes: PNPLA3 (I148M and K434E) TM6SF2 (E167K), and HFE (H63D and C282Y), based on undocumented case-control studies in the Saudi Arabian population. A total of 95 patients with NAFLD and 78 non-NAFLD subjects were recruited. Genomic DNA was isolated, and polymerase chain reaction and Sanger sequencing were performed using specific primers for the I148M, K434E, E167K, H63D, and C282Y. NAFLD subjects were older when compared to controls and showed the significant association (p = 0.0001). Non-significant association was found between gender (p = 0.26). However, both weight and BMI were found to be associated. Hardy-Weinberg equilibrium analysis confirmed that H63D, I148M, and K434E polymorphisms were associated. Genotype analysis showed only K434E variant was associated with NAFLD and non-NAFLD (OR-2.16; 95% CI: 1.08-4.31; p = 0.02). However, other polymorphisms performed with NAFLD and NASH were not associated (p &gt; 0.05), and similar analysis was found when ANOVA was performed (p &gt; 0.05). In conclusion, we confirmed that K434E polymorphism showed a positive association in the Saudi population.'},\n",
       " {'title': 'Decreased Hyocholic Acid and Lysophosphatidylcholine Induce Elevated Blood Glucose in a Transgenic Porcine Model of Metabolic Disease.',\n",
       "  'author': ['Xu J', 'Zhang K', 'Qiu B', 'Liu J', 'Liu X', 'Yang S', 'Xiao X'],\n",
       "  'abstract': '(1) Background: This work aims to investigate the metabolomic changes in PIGinH11 pigs and investigate differential compounds as potential therapeutic targets for metabolic diseases. (2) Methods: PIGinH11 pigs were established with a CRISPR/Cas9 system. PNPLA3(I148M), hIAPP, and GIPR(dn) were knocked in the H11 locus of the pig genome. The differential metabolites between and within groups were compared at baseline and two months after high-fat-high-sucrose diet induction. (3) Results: 72.02% of the 815 detected metabolites were affected by the transgenic effect. Significantly increased metabolites included isoleucine, tyrosine, methionine, oxoglutaric acid, acylcarnitine, glucose, sphinganines, ceramides, and phosphatidylserines, while fatty acids and conjugates, phosphatidylcholines, phosphatidylethanolamines, and sphingomyelins were decreased. Lower expression of GPAT3 and higher expression of PNPLA3(I148M) decreased the synthesis of diacylglycerol and phosphatidylcholines. Accumulated ceramides that block Akt signaling and decrease hyocholic acid and lysophosphatidylcholines might be the main reason for increased blood glucose in PIGinH11 pigs, which was consistent with metabolomic changes in patients. (4) Conclusions: Through serum metabolomics and lipidomics studies, significant changes in obesity and diabetes-related biomarkers were detected in PIGinH11 pigs. Excessive fatty acids beta-oxidation interfered with glucose and amino acids catabolism and reduced phosphatidylcholines. Decreased hyocholic acid, lysophosphatidylcholine, and increased ceramides exacerbated insulin resistance and elevated blood glucose. Phosphatidylserines were also increased, which might promote chronic inflammation by activating macrophages.'},\n",
       " {'title': 'MTARC1 and HSD17B13 Variants Have Protective Effects on Non-Alcoholic Fatty Liver Disease in Patients Undergoing Bariatric Surgery.',\n",
       "  'author': ['Kalinowski P',\n",
       "   'Smyk W',\n",
       "   'Nowosad M',\n",
       "   'Paluszkiewicz R',\n",
       "   'Michalowski L',\n",
       "   'Ziarkiewicz-Wroblewska B',\n",
       "   'Weber SN',\n",
       "   'Milkiewicz P',\n",
       "   'Lammert F',\n",
       "   'Zieniewicz K',\n",
       "   'Krawczyk M'],\n",
       "  'abstract': 'The severity of hepatic steatosis is modulated by genetic variants, such as patatin-like phospholipase domain containing 3 (PNPLA3) rs738409, transmembrane 6 superfamily member 2 (TM6SF2) rs58542926, and membrane-bound O-acyltransferase domain containing 7 (MBOAT7) rs641738. Recently, mitochondrial amidoxime reducing component 1 (MTARC1) rs2642438 and hydroxysteroid 17-beta dehydrogenase 13 (HSD17B13) rs72613567 polymorphisms were shown to have protective effects on liver diseases. Here, we evaluate these variants in patients undergoing bariatric surgery. A total of 165 patients who underwent laparoscopic sleeve gastrectomy and intraoperative liver biopsies and 314 controls were prospectively recruited. Genotyping was performed using TaqMan assays. Overall, 70.3% of operated patients presented with hepatic steatosis. NASH (non-alcoholic steatohepatitis) was detected in 28.5% of patients; none had cirrhosis. The increment of liver fibrosis stage was associated with decreasing frequency of the MTARC1 minor allele (p = 0.03). In multivariate analysis MTARC1 was an independent protective factor against fibrosis >/= 1b (OR = 0.52, p = 0.03) and >/= 1c (OR = 0.51, p = 0.04). The PNPLA3 risk allele was associated with increased hepatic steatosis, fibrosis, and NASH (OR = 2.22, p = 0.04). The HSD17B13 polymorphism was protective against liver injury as reflected by lower AST (p = 0.04) and ALT (p = 0.03) activities. The TM6SF2 polymorphism was associated with increased ALT (p = 0.04). In conclusion, hepatic steatosis is common among patients scheduled for bariatric surgery, but the MTARC1 and HSD17B13 polymorphisms lower liver injury in these individuals.'},\n",
       " {'title': 'Porphyra Haitanensis Polysaccharides Attenuates Blood Lipid via Gut-Liver Axis in Diet-Induced High-Fat Mesocricetus Auratus through Multiple Integrated Omics.',\n",
       "  'author': ['Zeng H', 'Chen P', 'Wang Z', 'Hu X', 'Zhang Y', 'Zheng B'],\n",
       "  'abstract': \"SCOPE: Hyperlipidemia is currently a global public health problem severely affecting people's physical and mental health, as well as their quality of life. METHODS AND RESULTS: The present study was aimed at revealing the mechanism of Porphyra haitanensis polysaccharide (PHP) in decreasing blood lipids by acting through gut-liver axis in Mesocricetus auratus fed a high-fat diet. PHP significantly prevented increases in serum total cholesterol, triglycerides and low-density lipoprotein cholesterol, and alleviated damage to liver cells induced by a high-fat diet Mesocricetus auratus, in a dose-dependent manner. PHP promoted proliferation of Muribaculaceae and Faecalibaculum, thereby enhancing the production of butyric acid both in the colon and liver, particular high-dose PHP (HPHP). Low-dose PHP (LPHP) promoted the expression of phosphatidylcholine metabolites and fatty acid transport genes, and inhibited the expression of genes involved in fat degradation (Abhd5), adipogenesis (Me1), fatty acid synthesis (Fasn and Pnpla3) and fatty acid chain elongation (Elovl6) in the liver. However, HPHP inhibited the expression of triglyceride metabolites and promoted the expression of fatty acid transporter (CD36), fatty acid oxidation (Acacb) and pparg genes in the liver. CONCLUSION: PHP regulates lipid metabolism through the gut microbiota, and the gut-liver axis plays an important role in its hypolipidemic effects. This article is protected by copyright. All rights reserved.\"},\n",
       " {'title': 'Risk factors in nonalcoholic fatty liver disease.',\n",
       "  'author': ['Ko E', 'Yoon EL', 'Jun DW'],\n",
       "  'abstract': 'Nonalcoholic fatty liver disease (NAFLD) is the most common liver disease, with a global prevalence estimated at approximately 25%. NAFLD is also the leading cause of liver cirrhosis, hepatocellular carcinoma, and death. Additionally, the risk of cardiovascular disease increases with greater NAFLD severity. The liver- and cardiovascular disease-related mortality incident rate ratios among the NAFLD population were 0.77 and 4.79 per 1,000 person-years, respectively. We intend to discuss the risk factors associated with NAFLD in terms of development and progression. Obesity or higher body mass index is closely associated with NAFLD in a dose-dependent manner, but growing evidence suggests that central obesity plays a more important role in the development of NAFLD. Saturated fat and fructose have been reported to be closely related to NAFLD. Fructose intake promotes lipogenesis and impairs mitochondria fat oxidation. The presence of type 2 diabetes is the most powerful predictive risk factor for hepatic fibrosis in patients with NAFLD. Single nucleotide polymorphism is not only associated with the prevalence of NAFLD but also associated with increased liver disease mortality. Obstructive sleep apnea, intestinal dysbiosis, and sarcopenia are associated with the development of NAFLD.'},\n",
       " {'title': 'RISK FACTORS FOR HEPATOCELLULAR CARCINOMA IN PATIENTS WITH NON-ALCOHOLIC FATTY LIVER DISEASE.',\n",
       "  'author': ['Cavalcante LN', 'Dezan MGF', 'Paz CLDSL', 'Lyra AC'],\n",
       "  'abstract': 'Non-alcoholic fatty liver disease is growing in worldwide prevalence and thus, is expected to have a higher number of NAFLD-related hepatocellular carcinoma (HCC) in the following years. This review describes the risk factors associated with HCC in NAFLD-patients. The presence of liver cirrhosis is the preponderant one. Male gender, PNPLA3 variants, diabetes, and obesity also appear to predispose to the development of HCC, even in non-cirrhotic subjects. Thus far, intensive lifestyle modifications, including glycemic control, and obesity treatment, are effective therapies for NAFLD/ non-alcoholic steatohepatitis and, therefore, probably, also for HCC. Some drugs that aimed at decreasing inflammatory activity and fibrosis, as well as obesity, were studied. Other data have suggested the possibility of HCC chemoprevention. So far, however, there is no definitive evidence for the routine utilization of these drugs. We hope, in the future, to be able to profile patients at higher risk of NAFLD-HCC and outline strategies for early diagnosis and prevention.'},\n",
       " {'title': 'Genetic Variation of SAMM50 Is Not an Independent Risk Factor for Alcoholic Hepatocellular Carcinoma in Caucasian Patients.',\n",
       "  'author': ['Nischalke HD',\n",
       "   'Schmalz F',\n",
       "   'Buch S',\n",
       "   'Fischer J',\n",
       "   'Moller C',\n",
       "   'Matz-Soja M',\n",
       "   'Kramer B',\n",
       "   'Langhans B',\n",
       "   'Kluners A',\n",
       "   'Soyka M',\n",
       "   'Stickel F',\n",
       "   'Nattermann J',\n",
       "   'Berg T',\n",
       "   'Strassburg CP',\n",
       "   'Lutz P'],\n",
       "  'abstract': 'Hepatocellular carcinoma (HCC) is a severe complication of advanced alcoholic liver disease, which is modulated by genetic predisposition. Identifying new genetic loci might improve screening. Genetic variation of SAMM50 was linked to HCC. We aimed to validate this finding in a large cohort of patients with advanced alcoholic liver disease (ALD). A large, well-characterised cohort of patients with alcoholic cirrhosis without (n = 674) and with (n = 386) HCC, as well as controls with HCC due to viral hepatitis (n = 134), controls with heavy alcohol abuse without liver disease (n = 266) and healthy subjects (n = 237), were genotyped for SAMM50 rs3827385 and rs3761472 and for PNPLA3 rs738409. Genotype frequencies were compared between patients with alcohol-associated cirrhosis with and without HCC by uni- and multivariate analysis. Minor variants in both SAMM50 rs3827385 and rs3761472 were significantly more frequent in patients with alcoholic HCC versus alcoholic cirrhosis and versus the control cohorts. An even stronger association was noted for PNPLA3 rs738409. The univariate analysis resulted in an odds ratio (OR) of 1.8 for carriers of at least one minor variant of SAMM50 rs3827385 and rs3761472 (each p &lt; 0.001), but this association was lost in multivariate analysis with age (OR 1.1/year), male sex (OR 3.2), diabetes (OR 1.9) and carriage of PNPLA3 148M (OR 2.1) remaining in the final model. Although minor variants of both SAMM50 loci are strongly associated with alcoholic HCC, this association is not independent of carriage of the well-known risk variant PNPLA3 148M.'},\n",
       " {'title': 'Metabolic-Associated Fatty Liver Disease, Hepatitis B Surface Antigen Seroclearance, and Long-Term Risk of Hepatocellular Carcinoma in Chronic Hepatitis B.',\n",
       "  'author': ['Yu MW', 'Lin CL', 'Liu CJ', 'Wu WJ', 'Hu JT', 'Huang YW'],\n",
       "  'abstract': 'The value of metabolic-associated fatty liver disease (MAFLD) and its ability to assess hepatocellular carcinoma (HCC) risk remains uncertain for chronic hepatitis B (CHB). We evaluated the impacts of MAFLD and its coincidental metabolic abnormalities and related genetic predisposition on HCC incidence and mortality outcomes in CHB. We analyzed data from 1453 HBsAg-positive men (median age = 49.2 years at baseline) from a cohort of civil servants recruited from 1989-1992. MAFLD was defined as hepatic steatosis on ultrasound with obesity, diabetes, or metabolic dysfunction at baseline. During follow-up (median = 19.3 years), 105 HCC events occurred. MAFLD was not associated with HCC (adjusted hazard ratio (aHR) = 1.02) but was associated with a higher HBsAg seroclearance rate (aHR = 1.43). In mediation analysis, HBsAg seroclearance driven by hepatic steatosis explained 31.6% of the association between MAFLD and HCC. Antiviral treatment or fatty liver disease-associated genetic variants did not influence the MAFLD-HCC association. In contrast, even after adjustment for MAFLD and the other metabolic abnormalities, diabetes (aHR = 2.28), obesity (aHR = 1.72), and metabolic dysfunction (aHR = 3.30) increased the risk of HCC (all p &lt; 0.030). The risk of HCC increased with the number of metabolic abnormalities (vs 0: aHR = 2.05 and 5.72 for 2 and >/= 3 metabolic abnormalities, respectively), and the cumulative effect of metabolic abnormalities was found across subgroups categorized by hepatic steatosis as well as in participants both with and without HBsAg seroclearance. In conclusion, MAFLD was not associated with increased HCC incidence in CHB. A more informative assessment of HCC risk can be obtained by taking into account the number of metabolic abnormalities.'},\n",
       " {'title': 'Genetics in non-alcoholic fatty liver disease: The role of risk alleles through the lens of immune response.',\n",
       "  'author': ['Sookoian S', 'Pirola CJ'],\n",
       "  'abstract': \"The knowledge of the genetic component of non-alcoholic fatty liver disease (NAFLD) has grown exponentially over the last 10-15 years. This review summarizes current evidence and the latest developments in the genetics of NAFLD and non-alcoholic steatohepatitis (NASH) from the immune system's perspective. Activation of innate and or adaptive immune response is an essential driver of NAFLD disease severity and progression. Lipid and immune pathways are crucial in the pathophysiology of NAFLD and NASH. Here, we highlight novel applications of genomic techniques, including single-cell sequencing and the genetics of gene expression, to elucidate the potential involvement of NAFLD/NASH-risk alleles in modulating immune system cells. Together, our focus is to provide an overview of the potential involvement of the NAFLD/NASH-related risk variants in mediating the immune-driven liver disease severity and diverse systemic pleiotropic effect/s.\"},\n",
       " {'title': 'PNPLA3 allele frequency has no impact on biliary bile acid composition or disease course in patients with primary sclerosing cholangitis.',\n",
       "  'author': ['Farkkila M',\n",
       "   'Kautiainen H',\n",
       "   'Tenca A',\n",
       "   'Jokelainen K',\n",
       "   'Arola J'],\n",
       "  'abstract': 'BACKGROUND AND AIMS: Primary sclerosing cholangitis (PSC) is a chronic inflammatory disease that leads to bile duct strictures, cholestasis, and biliary cirrhosis. PNPLA3 (patatin-like phospholipase domain containing 3), regulates cellular lipid synthesis by converting lysophosphatidic acid into phosphatidic acid. Isoleucine mutation to methionine at position 148 (I148M) causes a loss of this function. Only two studies, with contradictory results, have evaluated the role of PNPLA3 in PSC. The rs738409(G) variant of PNPLA3 has been associated with an increased risk for transplantation in male patients with dominant strictures (DS). The study aimed to evaluate the PNPLA3 allele frequency effect on the clinical outcomes, progression, and prognosis of PSC. Furthermore, we analyzed the impact of PNPLA3 on phospholipid and bile acid composition to evaluate the effect of the PNPLA3 status on UDCA response. PATIENTS AND METHODS: We recruited 560 patients prospectively and collected clinical and laboratory data as well as liver histology and imaging findings. PNPLA3 (CC, CG, GG) alleles were analyzed with TaqManTM. We also analyzed bile acids (BA), cholesterol and phospholipids and individual BA from a sample aspirated during endoscopic retrograde cholangiography (ERC). RESULTS: Among the recruited patients, 58.4%, 35.7% and 5.9% had the wild (CC), heterozygous (CG) and homozygous (GG) alleles, respectively. The PNPLA3 haplotype did not impact bile composition or individual BA. In addition, we found no differences in age at diagnosis, disease progression, liver fibrosis or survival between the cohorts. CONCLUSIONS: The PNPLA3 I148M variant had no significant impact on on bile composition, including UDCA content, clinical outcomes, progression of liver fibrosis, hepatobiliary cancer risk, liver transplantation, or overall survival.'},\n",
       " {'title': 'Artificial intelligence model with deep learning in nonalcoholic fatty liver disease diagnosis: genetic based artificial neural networks.',\n",
       "  'author': ['Ulger Y', 'Delik A'],\n",
       "  'abstract': 'Nonalcoholic fatty liver disease (NAFLD) is one of the most common causes of chronic liver disease in the world. The NAFLD spectrum includes simple steatosis, steatohepatitis, fibrosis, cirrhosis, and hepatocellular carcinoma (HCC). Genetic, nutritional factors, obesity, insulin resistance, gut microbiota are among the risk factors for NAFLD. The genetic variant Patatin-like phospholipase domain-containing protein 3 (PNPLA3) plays an important role in the development of a number of liver diseases ranging from steatosis, chronic hepatitis, cirrhosis and HCC. Due to the increase in the prevalence of NAFLD, new models are being developed with machine learning, deep learning, artificial neural network (ANN) algorithms in the field of artificial intelligence (AI) to determine low-cost, noninvasive diagnostic methods. Models developed with ANN from AI modules are important in order to examine biochemical and genomic information in detail in the diagnosis of NAFLD. The aim of this study is to develop a simple ANN model using biochemical and genotypic parameters in the diagnosis of NAFLD. A total of 300 patients followed up with the diagnosis of NAFLD and 100 controls were included in the study. The data set was divided into two as training and test set. Genotyping of PNPLA3 (CC, CG, GG) as genomic analysis was performed with real time PCR device. The algorithm used for the diagnosis of NAFLD was designed using age, body mass index (BMI), mean platelet volume (MPV), insulin resistance (IR), alanine aminotransferase (ALT), genotype PNPLA3 (CC, CG, GG) parameters. MLP Classifier algorithm from ANN was used in the development of the model. ANN algorithms are used in python programming language. Statistical analyzes were made in SPSS program. Percent accuracy, area under the ROC curve, confusion matrix, Positive (PPV) and Negative Predicted Value (NPV) values, precision, recall, and f1-score results were determined. The accuracy percentage was determined as 0.979 in the train set and 0.970 in the test set. The Log Loss value was set to 0.09. The developed neural network achieved an accuracy percentage of 97.0% during testing, with an area under the ROC curve value of 0.95. We think that the ANN model developed with genomic and biochemical parameters can be used as a cost-effective, noninvasive new predictive diagnostic model in clinical practice in the diagnosis of NAFLD.'},\n",
       " {'title': 'Contribution of PNPLA3 gene polymorphisms to hepatocellular carcinoma susceptibility in the Chinese Han population.',\n",
       "  'author': ['Gong D', 'Li S', 'Yu Z', 'Wang K', 'Qiao X', 'Wu C'],\n",
       "  'abstract': 'OBJECTIVES: The purpose of this study was to investigate the association of PNPLA3 single nucleotide polymorphisms (SNPs) (rs738409 C > G, rs3747207 G > A, rs4823173 G > A, and rs2896019 T > G) with hepatocellular carcinoma (HCC) susceptibility. METHODS: This case-control study included 484 HCC patients and 487 controls. Logistic regression analysis was performed to study the associations of PNPLA3 gene polymorphisms with HCC susceptibility, and odds ratios with their corresponding 95% confidence intervals were calculated to evaluate these correlations. RESULTS: In the overall analysis, we found that the G allele (OR = 1.25, 95% CI = 1.04-1.50, p = 0.018, false discovery rate (FDR)-p = 0.035) and GG genotype (OR = 1.59, 95% CI = 1.06-2.39, p = 0.024, FDR-p = 0.048) of rs2896019 were significantly associated with increased HCC susceptibility. In stratified analysis, we found that all four SNPs were related to increased HCC susceptibility in subjects aged > 55 years. In haplotype analysis, the GAAG haplotype was significantly associated with increased HCC susceptibility (OR = 1.25, 95% CI = 1.03-1.53, p = 0.023, FDR-p = 0.046). Besides, we noticed that rs738409 was significantly correlated with alpha-fetoprotein (AFP) (p = 0.007), and HCC patients with the GG genotype had a higher level of AFP. CONCLUSIONS: Our study suggested that PNPLA3-rs2896019 was significantly associated with an increased susceptibility to HCC.'},\n",
       " {'title': 'Integrating genetic variants into clinical models for hepatocellular carcinoma risk stratification in cirrhosis.',\n",
       "  'author': ['Nahon P',\n",
       "   'Bamba-Funck J',\n",
       "   'Layese R',\n",
       "   'Trepo E',\n",
       "   'Zucman-Rossi J',\n",
       "   'Cagnot C',\n",
       "   'Ganne-Carrie N',\n",
       "   'Chaffaut C',\n",
       "   'Guyot E',\n",
       "   'Ziol M',\n",
       "   'Sutton A',\n",
       "   'Audureau E'],\n",
       "  'abstract': \"BACKGROUND AND AIMS: This study aimed to evaluate the ability of single nucleotide polymorphisms (SNPs) to refine hepatocellular carcinoma (HCC) risk stratification. METHODS: Six SNPs in PNPLA3, TM6SF2, HSD17B13, APOE, and MBOAT7 affecting lipid turnover and one variant involved in the Wnt-beta-catenin pathway (WNT3A-WNT9A rs708113) were assessed in patients with alcohol-related and/or HCV-cured cirrhosis included in HCC surveillance programs (prospective CirVir and CIRRAL cohorts). Their prognostic value for HCC occurrence was assessed using Fine-Gray models combined into a 7-SNP genetic risk score (GRS). Prediction ability of two clinical scores (a routine nongenetic model determined by multivariate analysis and the external aMAP score) without then with the addition of the GRS was evaluated by C-indices. The standardized net benefit was derived from decision curves. RESULTS: Among 1145 patients, 86 (7.5%) developed HCC after 43.7 months. PNPLA3 and WNT3A-WNT9A variants were independently associated with HCC occurrence. The GRS stratified the population into 3 groups with progressively increased 5-yr HCC incidence [Group 1 (n=627, 5.4%), Group 2 (n=276, 10.7%), and Group 3 (n=242, 15.3%); P<0.001]. The multivariate model identified age, male sex, diabetes, platelet count, GGT levels, albuminemia and the GRS as independent risk factors. The clinical model performance for 5-yr HCC prediction was similar to that of the aMAP score (C-Index 0.769). The addition of the GRS to both scores modestly improved their performance (C-Index 0.786 and 0.783, respectively). This finding was confirmed by decision curve analyses showing only fair clinical net benefit. CONCLUSIONS: Patients with cirrhosis can be stratified into HCC risk classes by variants affecting lipid turnover and Wnt-beta-catenin pathway. The incorporation of this genetic information modestly improves the performance of clinical scores. IMPACT AND IMPLICATIONS: The identification of patients at higher probability of developing liver cancer is pivotal to improve the performance of surveillance. Risk assessment can be achieved by combining several clinical and biological parameters used in routine practice. The addition of patients' genetic characteristics can modestly improve this prediction and will ultimately pave the way for precision medicine in patients eligible for HCC surveillance, allowing physicians to trigger personalized screening strategies.\"},\n",
       " {'title': 'Causes and Risk Profiles of Mortality among Individuals with Nonalcoholic Fatty Liver.',\n",
       "  'author': ['Konyn P', 'Ahmed A', 'Kim D'],\n",
       "  'abstract': 'Nonalcoholic fatty liver disease (NAFLD) is the most common cause of chronic liver disease in the United States and worldwide. Though nonalcoholic fatty liver per se may not be independently associated with an increased risk for all-cause mortality, it is associated with a number of harmful metabolic risk factors, such as type 2 diabetes mellitus, hyperlipidemia, obesity, a sedentary lifestyle, and an unhealthy diet. The fibrosis stage is a predictor of all-cause mortality in NAFLD. Mortality in individuals with NAFLD has been steadily increasing, and the most common cause-specific mortality for NAFLD is cardiovascular disease, followed by extra-hepatic cancer, liver-related mortality, and diabetes. High-risk profiles for mortality in NAFLD include PNPLA3 I148M polymorphism, low thyroid function and hypothyroidism, and sarcopenia. Achieving weight loss through adherence to a high-quality diet and sufficient physical activity is the most important predictor of improvement in NAFLD severity and the benefit of survival. Given the increasing health burden of NAFLD, future studies with more long-term mortality data may demonstrate an independent association between NAFLD and mortality.'},\n",
       " {'title': 'Different effects of low muscle mass on the risk of non-alcoholic fatty liver disease and hepatic fibrosis in a prospective cohort.',\n",
       "  'author': ['Choe HJ', 'Lee H', 'Lee D', 'Kwak SH', 'Koo BK'],\n",
       "  'abstract': 'BACKGROUND: Non-alcoholic fatty liver disease (NAFLD) and sarcopenia share insulin resistance as a common pathophysiology and have overlapping clinical manifestation of metabolic derangement; hence, it is difficult to differentiate the independent effect of sarcopenia on the development of NAFLD from concomitant metabolic disorders. Using a community-based prospective cohort study, the contributions of low muscle mass and genetic risk factors to the development of NAFLD and NAFLD-related hepatic fibrosis were investigated in the Korean population. METHODS: This prospective community-based cohort study included 40-70-year-old adults, followed up biennially from 2001-2002 to 2017-2018. NAFLD was defined as a hepatic steatosis index of >/=36, and hepatic fibrosis was defined based on the fibrosis-4 index. Sex-specific quartiles of body mass index (BMI)-adjusted muscle mass were calculated (muscle mass/BMI), and low muscle mass was defined as the lowest quartile (Q1). Cox proportional hazard models for incident NAFLD or hepatic fibrosis incorporating age, sex, BMI of >/=25 kg/m(2) , metabolic syndrome and PNPLA3 and TM6SF2 risk alleles were used to assess the independent determinants for incident NAFLD and hepatic fibrosis among individuals with NAFLD at baseline. RESULTS: Among the 4038 participants without NAFLD at baseline (mean age, 51.5 +/- 8.8 years), 920 (22.8%) developed NAFLD during the 12-year follow-up period. As muscle mass decreased, the risk of NAFLD increased even after adjustment for age, sex, obesity, metabolic syndrome and PNPLA3 and TM6SF2 risk alleles [hazard ratio (HR) per quartile, 1.18, 95% confidence interval (CI), 1.11-1.27, P < 0.001]. TM6SF2 also affected the risk of NAFLD development [HR 1.19, (95% CI, 1.00-1.40), P = 0.044]. Of the 1176 patients with NAFLD but without hepatic fibrosis at baseline, the incident of hepatic fibrosis was found in 51.8%, 44.7%, 42.6% and 41.0% in Q1, Q2, Q3 and Q4 of BMI-adjusted muscle mass, respectively, during the follow-up period (P for trend = 0.006). However, this trend lost its statistical significance when adjusted for confounders. The PNPLA3 risk variant, but not the TM6SF2 genotype, was an independent risk factor for developing hepatic fibrosis among patients with NAFLD (HR 1.17, 95% CI 1.04-1.32, P = 0.010). CONCLUSIONS: Both lower muscle mass index and genetic risk variants are important contributors to the development of NAFLD. In patients already diagnosed with NAFLD, however, PNPLA3 confers a greater risk for hepatic fibrosis progression than lower muscle mass.'},\n",
       " {'title': 'Genetic variants associated with circulating liver injury markers in Mexican Americans, a population at risk for non-alcoholic fatty liver disease.',\n",
       "  'author': ['Sabotta CM',\n",
       "   'Kwan SY',\n",
       "   'Petty LE',\n",
       "   'Below JE',\n",
       "   'Joon A',\n",
       "   'Wei P',\n",
       "   'Fisher-Hoch SP',\n",
       "   'McCormick JB',\n",
       "   'Beretta L'],\n",
       "  'abstract': 'Objective: Mexican Americans are disproportionally affected by non-alcoholic fatty liver disease (NAFLD), liver fibrosis and hepatocellular carcinoma. Noninvasive means to identify those in this population at high risk for these diseases are urgently needed. Approach: The Cameron County Hispanic Cohort (CCHC) is a population-based cohort with high rates of obesity (51%), type 2 diabetes (28%) and NAFLD (49%). In a subgroup of 564 CCHC subjects, we evaluated 339 genetic variants previously reported to be associated with liver injury markers aspartate aminotransferase (AST) and alanine aminotransferase (ALT) in United Kingdom and Japanese cohorts. Results: Association was confirmed for 86 variants. Among them, 27 had higher effect allele frequency in the CCHC than in the United Kingdom and Japanese cohorts, and 16 had stronger associations with AST and ALT than rs738409 (PNPLA3). These included rs17710008 (MYCT1), rs2519093 (ABO), rs1801690 (APOH), rs10409243 (S1PR2), rs1800759 (LOC100507053) and rs2491441 (RGL1), which were also associated with steatosis and/or liver fibrosis measured by vibration-controlled transient elastography. Main contributors to advanced fibrosis risk were rs11240351 (CNTN2), rs1800759 (LOC100507053), rs738409 (PNPLA3) and rs1801690 (APOH), with advanced fibrosis detected in 37.5% of subjects with 3 of these 4 variants [AOR = 11.6 (95% CI) = 3.8-35.3]. AST- and ALT-associated variants implicated distinct pathways (ethanol and galactose degradation versus antigen presentation and B cell development). Finally, 8 variants, including rs62292950 (DNAJC13), were associated with gut microbiome changes. Conclusion: These genotype-phenotype findings may have utility in risk modeling and disease prevention in this high-risk population.'}]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "result_PNPLA3=extrair_info(\"PNPLA3\",\"...@gmail.com\" )\n",
    "gravarBD(result_PNPLA3, \"informaçao PNPLA3\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "32fdbf0c",
   "metadata": {},
   "source": [
    "# Para o TFAP2B"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "4602b7f4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'title': \"KCTD1 and Scalp-Ear-Nipple ('Finlay-Marks') syndrome may be associated with myopia and Thin basement membrane nephropathy through an effect on the collagen IV alpha3 and alpha4 chains.\",\n",
       "  'author': ['Wang D',\n",
       "   'Trevillian P',\n",
       "   'May S',\n",
       "   'Diakumis P',\n",
       "   'Wang Y',\n",
       "   'Colville D',\n",
       "   'Bahlo M',\n",
       "   'Greferath U',\n",
       "   'Fletcher E',\n",
       "   'Young B',\n",
       "   'Mack HG',\n",
       "   'Savige J'],\n",
       "  'abstract': 'INTRODUCTION: Scalp-Ear-Nipple syndrome is caused by pathogenic KCTD1 variants and characterised by a scalp defect, prominent ears, and rudimentary breasts. We describe here further clinical associations in the eye and kidney. METHODS: Fifteen affected members from two unrelated families with p.(Ala30Glu) or p.(Pro31Leu) in KCTD1 were examined for ocular and renal abnormalities. The relevant proteins were studied in the eye and kidney, and the mutation consequences determined from mouse knockout models. RESULTS: Five males and 10 females with a median age of 40 years (range 1-70) with pathogenic variants p.(Ala30Glu) (n = 12) or p.(Pro31Leu) (n = 3) in KCTD1 were studied. Of the 6 who underwent detailed ophthalmic examination, 5 (83%) had low myopic astigmatism, the mean spherical equivalent of 10 eyes was 2.38D, and one (17%) had hypermetropic astigmatism. One female had a divergent strabismus.Five individuals had renal cysts (5/15, 33%), with renal biopsy in one demonstrating a thinned glomerular basement membrane identical to that seen in Thin basement membrane nephropathy (AD Alport syndrome).In the eye, KCTD1 and its downstream targets, TFAP2, and the collagen IV alpha3 and alpha4 chains localised to the cornea and near the retinal amacrine cells. In the kidney, all these proteins except TFAP2 were expressed in the podocytes and distal tubules. TFAP2B and COL4A4 knockout mice also had kidney cysts, and COL4A3 and COL4A4 knockout mice had myopia. CONCLUSION: Individuals with a pathogenic KCTD1 variant may have low myopic astigmatism and represent a further rare genetic cause for a thinned glomerular basement membrane.'},\n",
       " {'title': 'Urine Molecular Biomarkers for Detection and Follow-Up of Small Renal Masses.',\n",
       "  'author': ['Zalimas A',\n",
       "   'Kubiliute R',\n",
       "   'Zukauskaite K',\n",
       "   'Sabaliauskaite R',\n",
       "   'Trakymas M',\n",
       "   'Letautiene S',\n",
       "   'Kaubriene EM',\n",
       "   'Usinskiene J',\n",
       "   'Ulys A',\n",
       "   'Jarmalaite S'],\n",
       "  'abstract': \"Active surveillance (AS) is the best strategy for small renal masses (SRMs) management; however, reliable methods for early detection and disease aggressiveness prediction are urgently needed. The aim of the present study was to validate DNA methylation biomarkers for non-invasive SRM detection and prognosis. The levels of methylated genes TFAP2B, TAC1, PCDH8, ZNF677, FLRT2, and FBN2 were evaluated in 165 serial urine samples prospectively collected from 39 patients diagnosed with SRM, specifically renal cell carcinoma (RCC), before and during the AS via quantitative methylation-specific polymerase chain reaction. Voided urine samples from 92 asymptomatic volunteers were used as the control. Significantly higher methylated TFAP2B, TAC1, PCDH8, ZNF677, and FLRT2 levels and/or frequencies were detected in SRM patients' urine samples as compared to the control. The highest diagnostic power (AUC = 0.74) was observed for the four biomarkers panel with 92% sensitivity and 52% specificity. Methylated PCDH8 level positively correlated with SRM size at diagnosis, while TFAP2B had the opposite effect and was related to SRM progression. To sum up, SRMs contribute significantly to the amount of methylated DNA detectable in urine, which might be used for very early RCC detection. Moreover, PCDH8 and TFAP2B methylation have the potential to be prognostic biomarkers for SRMs.\"},\n",
       " {'title': 'Association between genetic risk score and tri-ponderal mass index growth trajectories among different dietary consumption adolescents in a prospective Taiwanese cohort.',\n",
       "  'author': ['Wu YF', 'Chien KL', 'Chen YC'],\n",
       "  'abstract': 'BACKGROUND: Single-nucleotide polymorphisms (SNPs) in various genetic loci are associated with childhood obesity; however, their influence on adolescent growth patterns has rarely been explored. This study investigated whether genetic variants could predict tri-ponderal mass index (TMI)-derived growth trajectories and the interaction between genetic and dietary factors. METHODS: We conducted Taiwan Puberty Longitudinal Study, a prospective cohort that recruited 1,135 children since 2018. Anthropometric measurements were recorded every three months, while dietary nutrition assessment and biological sampling for genotyping were collected during the first visit. TMI growth trajectory groups were identified using growth mixture modeling. A multinomial logistic regression model for different growth trajectories was used to examine the effect of candidate SNPs, and the most related SNPs were used to establish the genetic risk score. We then explored the effect of the genetic risk score in subgroup analysis according to dietary calories and different dietary consumption patterns. RESULTS: Three TMI-based growth trajectory groups were identified among adolescents. The \"increased weight\" trajectory group accounted for approximately 9.7% of the participants. FTO/rs7206790 was associated with the increased weight growth trajectory after adjusting for the baseline TMI and other correlated covariates (OR: 2.13, 95% CI: 1.08-4.21). We generated the genetic risk score using 4 SNPs (FTO/rs7206790, ADCY9/rs2531995, TFAP2B/rs4715210, and TMEM18/rs6548238) and selected the threshold of 10 points to define risk categories. There were 11.66% and 3.24% of participants belonged to the increased weight trajectory in high- and low-risk groups, respectively; and the predictive ability of the genetic risk score was notable among low calories intake participants (OR: 1.90, 95% CI: 1.18-3.05 vs. OR: 1.17, 95% CI: 0.78-1.75 in high calories intake group). CONCLUSION: Our results offer a new perspective on the genetic and dietary basis of changes in adolescent obesity status. Individualized interventions for obesity prevention may be considered among high-risk children.'},\n",
       " {'title': 'DRAGON: Determining Regulatory Associations using Graphical models on multi-Omic Networks.',\n",
       "  'author': ['Shutta KH',\n",
       "   'Weighill D',\n",
       "   'Burkholz R',\n",
       "   'Guebila MB',\n",
       "   'DeMeo DL',\n",
       "   'Zacharias HU',\n",
       "   'Quackenbush J',\n",
       "   'Altenbuchinger M'],\n",
       "  'abstract': \"The increasing quantity of multi-omic data, such as methylomic and transcriptomic profiles collected on the same specimen or even on the same cell, provides a unique opportunity to explore the complex interactions that define cell phenotype and govern cellular responses to perturbations. We propose a network approach based on Gaussian Graphical Models (GGMs) that facilitates the joint analysis of paired omics data. This method, called DRAGON (Determining Regulatory Associations using Graphical models on multi-Omic Networks), calibrates its parameters to achieve an optimal trade-off between the network's complexity and estimation accuracy, while explicitly accounting for the characteristics of each of the assessed omics 'layers.' In simulation studies, we show that DRAGON adapts to edge density and feature size differences between omics layers, improving model inference and edge recovery compared to state-of-the-art methods. We further demonstrate in an analysis of joint transcriptome - methylome data from TCGA breast cancer specimens that DRAGON can identify key molecular mechanisms such as gene regulation via promoter methylation. In particular, we identify Transcription Factor AP-2 Beta (TFAP2B) as a potential multi-omic biomarker for basal-type breast cancer. DRAGON is available as open-source code in Python through the Network Zoo package (netZooPy v0.8; netzoo.github.io).\"},\n",
       " {'title': 'Changes in Novel Anthropometric Indices of Abdominal Obesity during Weight Loss with Selected Obesity-Associated Single-Nucleotide Polymorphisms: A Small One-Year Pilot Study.',\n",
       "  'author': ['Ilowiecka K', 'Glibowski P', 'Libera J', 'Koch W'],\n",
       "  'abstract': 'Whether BMI and the competing waist circumference (WC)-based anthropometric indices are associated with obesity-related single-nucleotide polymorphisms (SNPs) is as yet unknown. The current study aimed to evaluate the anthropometric indices (fat mass index, body shape index, visceral adiposity index, relative fat mass, body roundness index, and conicity index) during a weight loss intervention in 36 obese individuals. Blood biochemical parameters (total cholesterol, low-density lipoprotein, high-density lipoprotein, and triglycerides) and three SNPs (FTO rs9939609, TFAP2B rs987237, and PLIN1 rs894160) were assessed in 22 women and 14 men (35.58 +/- 9.85 years, BMI 35.04 +/- 3.80 kg/m(2)) who completed a 12-month balanced energy-restricted diet weight loss program. Body composition was assessed via bioelectrical impedance (SECA mBCA515). At the end of the weight loss intervention, all anthropometric indices were significantly reduced (p &lt; 0.05). For the SNP FTO rs9939609, the higher risk allele (A) was characteristic of 88.9% of the study group, in which 10 participants (27.8%) were homozygous. We found a similar distribution of alleles in TFAP2B and PLIN1. Heterozygous genotypes in FTO rs9939609 and TFAP2B rs987237 were predisposed to significant reductions in WC-based novel anthropometric indices during weight loss. The influence of PLIN1 rs894160 polymorphisms on the changes in the analyzed indices during weight loss has not been documented in the present study.'},\n",
       " {'title': 'TFAP2B Haploinsufficiency Impacts Gastrointestinal Function and Leads to Pediatric Intestinal Pseudo-obstruction.',\n",
       "  'author': ['Zada A',\n",
       "   'Kuil LE',\n",
       "   'de Graaf BM',\n",
       "   'Kakiailatu N',\n",
       "   'Windster JD',\n",
       "   'Brooks AS',\n",
       "   'van Slegtenhorst M',\n",
       "   'de Koning B',\n",
       "   'Wijnen RMH',\n",
       "   'Melotte V',\n",
       "   'Hofstra RMW',\n",
       "   'Brosens E',\n",
       "   'Alves MM'],\n",
       "  'abstract': 'Background: Pediatric Intestinal Pseudo-obstruction (PIPO) is a congenital enteric disorder characterized by severe gastrointestinal (GI) dysmotility, without mechanical obstruction. Although several genes have been described to cause this disease, most patients do not receive a genetic diagnosis. Here, we aim to identify the genetic cause of PIPO in a patient diagnosed with severe intestinal dysmotility shortly after birth. Methods: Whole exome sequencing (WES) was performed in the patient and unaffected parents, in a diagnostic setting. After identification of the potential disease-causing variant, its functional consequences were determined in vitro and in vivo. For this, expression constructs with and without the causing variant, were overexpressed in HEK293 cells. To investigate the role of the candidate gene in GI development and function, a zebrafish model was generated where its expression was disrupted using CRISPR/Cas9 editing. Results: WES analysis identified a de novo heterozygous deletion in TFAP2B (NM_003221.4:c.602-5_606delTCTAGTTCCA), classified as a variant of unknown significance. In vitro studies showed that this deletion affects RNA splicing and results in loss of exon 4, leading to the appearance of a premature stop codon and absence of TFAP2B protein. Disruption of tfap2b in zebrafish led to decreased enteric neuronal numbers and delayed transit time. However, no defects in neuronal differentiation were detected. tfap2b crispants also showed decreased levels of ednrbb mRNA, a downstream target of tfap2b. Conclusion: We showed that TFAP2B haploinsufficiency leads to reduced neuronal numbers and GI dysmotility, suggesting for the first time, that this gene is involved in PIPO pathogenesis.'},\n",
       " {'title': 'TRIM37 Augments AP-2gamma Transcriptional Activity and Cellular Localization via K63-linked Ubiquitination to Drive Breast Cancer Progression.',\n",
       "  'author': ['Cui G',\n",
       "   'Gao Z',\n",
       "   'Chang S',\n",
       "   'Narwade N',\n",
       "   'Chen Y',\n",
       "   'Poudel B',\n",
       "   'Lei KMK',\n",
       "   'Zhang W',\n",
       "   'Li G',\n",
       "   'Poon TCW',\n",
       "   'Cheung E'],\n",
       "  'abstract': 'Activator Protein 2 gamma (AP-2gamma) is a master transcription factor that plays a critical role in the development and progression of breast cancer. However, the underlying mechanism is still unclear. Herein, using a proteomics approach, we identified Tripartite motif-containing 37 (TRIM37) as a novel coactivator of AP-2gamma-mediated transcription in breast cancer cells. We demonstrate that TRIM37 facilitates AP-2gamma chromatin binding to directly regulate the AP-2gamma mediated transcriptional program. We also show that TRIM37 achieves this by stimulating K63 chain-linked ubiquitination of AP-2gamma, promoting protein localization from the cytoplasm to the nucleus. In clinical analyses, we find TRIM37 is upregulated in multiple breast cancer datasets, supporting our findings that the TRIM37-AP-2gamma interaction is essential for breast cancer tumor growth. Overall, our work reveals that TRIM37 is an oncogenic coactivator of AP-2gamma in breast cancer and provides a novel therapeutic target for treating the disease.'},\n",
       " {'title': 'Integrative RNA profiling of TBEV-infected neurons and astrocytes reveals potential pathogenic effectors.',\n",
       "  'author': ['Selinger M',\n",
       "   'Vechtova P',\n",
       "   'Tykalova H',\n",
       "   'Oslejskova P',\n",
       "   'Rumlova M',\n",
       "   'Sterba J',\n",
       "   'Grubhoffer L'],\n",
       "  'abstract': 'Tick-borne encephalitis virus (TBEV), the most medically relevant tick-transmitted flavivirus in Eurasia, targets the host central nervous system and frequently causes severe encephalitis. The severity of TBEV-induced neuropathogenesis is highly cell-type specific and the exact mechanism responsible for such differences has not been fully described yet. Thus, we performed a comprehensive analysis of alterations in host poly-(A)/miRNA/lncRNA expression upon TBEV infection in vitro in human primary neurons (high cytopathic effect) and astrocytes (low cytopathic effect). Infection with severe but not mild TBEV strain resulted in a high neuronal death rate. In comparison, infection with either of TBEV strains in human astrocytes did not. Differential expression and splicing analyses with an in silico prediction of miRNA/mRNA/lncRNA/vd-sRNA networks found significant changes in inflammatory and immune response pathways, nervous system development and regulation of mitosis in TBEV Hypr-infected neurons. Candidate mechanisms responsible for the aforementioned phenomena include specific regulation of host mRNA levels via differentially expressed miRNAs/lncRNAs or vd-sRNAs mimicking endogenous miRNAs and virus-driven modulation of host pre-mRNA splicing. We suggest that these factors are responsible for the observed differences in the virulence manifestation of both TBEV strains in different cell lines. This work brings the first complex overview of alterations in the transcriptome of human astrocytes and neurons during the infection by two TBEV strains of different virulence. The resulting data could serve as a starting point for further studies dealing with the mechanism of TBEV-host interactions and the related processes of TBEV pathogenesis.'},\n",
       " {'title': 'Allele-specific expression reveals genes with recurrent cis-regulatory alterations in high-risk neuroblastoma.',\n",
       "  'author': ['Sen A', 'Huo Y', 'Elster J', 'Zage PE', 'McVicker G'],\n",
       "  'abstract': 'BACKGROUND: Neuroblastoma is a pediatric malignancy with a high frequency of metastatic disease at initial diagnosis. Neuroblastoma tumors have few recurrent protein-coding mutations but contain extensive somatic copy number alterations (SCNAs) suggesting that mutations that alter gene dosage are important drivers of tumorigenesis. Here, we analyze allele-specific expression in 96 high-risk neuroblastoma tumors to discover genes impacted by cis-acting mutations that alter dosage. RESULTS: We identify 1043 genes with recurrent, neuroblastoma-specific allele-specific expression. While most of these genes lie within common SCNA regions, many of them exhibit allele-specific expression in copy neutral samples and these samples are enriched for mutations that are predicted to cause nonsense-mediated decay. Thus, both SCNA and non-SCNA mutations frequently alter gene expression in neuroblastoma. We focus on genes with neuroblastoma-specific allele-specific expression in the absence of SCNAs and find 26 such genes that have reduced expression in stage 4 disease. At least two of these genes have evidence for tumor suppressor activity including the transcription factor TFAP2B and the protein tyrosine phosphatase PTPRH. CONCLUSIONS: In summary, our allele-specific expression analysis discovers genes that are recurrently dysregulated by both large SCNAs and other cis-acting mutations in high-risk neuroblastoma.'},\n",
       " {'title': 'Prdm6 controls heart development by regulating neural crest cell differentiation and migration.',\n",
       "  'author': ['Hong L',\n",
       "   'Li N',\n",
       "   'Gasque V',\n",
       "   'Mehta S',\n",
       "   'Ye L',\n",
       "   'Wu Y',\n",
       "   'Li J',\n",
       "   'Gewies A',\n",
       "   'Ruland J',\n",
       "   'Hirschi KK',\n",
       "   'Eichmann A',\n",
       "   'Hendry C',\n",
       "   'van Dijk D',\n",
       "   'Mani A'],\n",
       "  'abstract': 'The molecular mechanisms that drive the acquisition of distinct neural crest cell (NCC) fates is still poorly understood. Here, we identified Prdm6 as an epigenetic modifier that temporally and spatially regulates the expression of NCC specifiers and determines the fate of a subset of migrating cardiac NCCs (CNCCs). Using transcriptomic analysis and genetic and fate mapping approaches in transgenic mice, we showed that disruption of Prdm6 was associated with impaired CNCC differentiation, delamination, and migration and led to patent ductus arteriosus (DA) and ventricular noncompaction. Bulk and single-cell RNA-Seq analyses of the DA and CNCCs identified Prdm6 as a regulator of a network of CNCC specification genes, including Wnt1, Tfap2b, and Sox9. Loss of Prdm6 in CNCCs diminished its expression in the pre-epithelial-mesenchymal transition (pre-EMT) cluster, resulting in the retention of NCCs in the dorsal neural tube. This defect was associated with diminished H4K20 monomethylation and G1-S progression and augmented Wnt1 transcript levels in pre-EMT and neural tube clusters, which we showed was the major driver of the impaired CNCC migration. Altogether, these findings revealed Prdm6 as a key regulator of CNCC differentiation and migration and identified Prdm6 and its regulated network as potential targets for the treatment of congenital heart diseases.'},\n",
       " {'title': 'Tfap2b specifies an embryonic melanocyte stem cell that retains adult multifate potential.',\n",
       "  'author': ['Brombin A',\n",
       "   'Simpson DJ',\n",
       "   'Travnickova J',\n",
       "   'Brunsdon H',\n",
       "   'Zeng Z',\n",
       "   'Lu Y',\n",
       "   'Young AIJ',\n",
       "   'Chandra T',\n",
       "   'Patton EE'],\n",
       "  'abstract': 'Melanocytes, the pigment-producing cells, are replenished from multiple stem cell niches in adult tissue. Although pigmentation traits are known risk factors for melanoma, we know little about melanocyte stem cell (McSC) populations other than hair follicle McSCs and lack key lineage markers with which to identify McSCs and study their function. Here we find that Tfap2b and a select set of target genes specify an McSC population at the dorsal root ganglia in zebrafish. Functionally, Tfap2b is required for only a few late-stage embryonic melanocytes, and is essential for McSC-dependent melanocyte regeneration. Fate mapping data reveal that tfap2b(+) McSCs have multifate potential, and are the cells of origin for large patches of adult melanocytes, two other pigment cell types (iridophores and xanthophores), and nerve-associated cells. Hence, Tfap2b confers McSC identity in early development, distinguishing McSCs from other neural crest and pigment cell lineages, and retains multifate potential in the adult zebrafish.'},\n",
       " {'title': 'Clinical significance of novel DNA methylation biomarkers for renal clear cell carcinoma.',\n",
       "  'author': ['Kubiliute R',\n",
       "   'Zukauskaite K',\n",
       "   'Zalimas A',\n",
       "   'Ulys A',\n",
       "   'Sabaliauskaite R',\n",
       "   'Bakavicius A',\n",
       "   'Zelvys A',\n",
       "   'Jankevicius F',\n",
       "   'Jarmalaite S'],\n",
       "  'abstract': 'OBJECTIVE: Clear cell renal cell carcinoma (ccRCC) is the most common type of kidney tumor characterized by the highest mortality rate of the genitourinary cancers, and, therefore, new diagnostic and/or prognostic biomarkers are urgently needed. METHODS: Based on genome-wide DNA methylation profiling in 11 pairs of ccRCC and non-cancerous renal tissues (NRT), the methylation at regulatory regions of ZNF677, FBN2, PCDH8, TFAP2B, TAC1, and FLRT2 was analyzed in 168 renal tissues and 307 urine samples using qualitative and quantitative methylation-specific PCR (MSP). RESULTS: Significantly higher methylation frequencies for all genes were found in ccRCC tissues compared to NRT (33-60% vs. 0-11%). The best diagnostic performance demonstrated a panel of ZNF677, FBN2, PCDH8, TFAP2B & TAC1 with 82% sensitivity and 96% specificity. Hypermethylation of ZNF677 and PCDH8 in the tissue samples was significantly related to numerous adverse clinicopathologic parameters. For the urine-based ccRCC detection, the highest diagnostic power (AUC = 0.78) was observed for a panel of ZNF677 & PCDH8 (with or without FBN2 or FLRT2) with 69-78% sensitivity and 69-80% specificity, albeit with lower values in the validation cohort. Besides, methylation of PCDH8 was significantly related to higher tumor stage and fat invasion in the study and validation cohorts. Moreover, PCDH8 was strongly predictive for OS (HR, 5.7; 95% CI 1.16-28.12), and its prognostic power considerably increased in combination with ZNF677 (HR, 12.5; 95% CI 1.47-105.58). CONCLUSION: In summary, our study revealed novel, potentially promising DNA methylation biomarkers of ccRCC with the possibility to be applied for non-invasive urine-based ccRCC detection and follow-up.'},\n",
       " {'title': 'Heterogeneous miRNA-mRNA Regulatory Networks of Visceral and Subcutaneous Adipose Tissue in the Relationship Between Obesity and Renal Clear Cell Carcinoma.',\n",
       "  'author': ['Liu Y', 'Liu Y', 'Hu J', 'He Z', 'Liu L', 'Ma Y', 'Wen D'],\n",
       "  'abstract': 'BACKGROUND: Clear cell renal cell carcinoma (ccRCC) is one of the most lethal urologic cancer. Associations of both visceral adipose tissue (VAT) and subcutaneous adipose tissue (SAT) with ccRCC have been reported, and underlying mechanisms of VAT perhaps distinguished from SAT, considering their different structures and functions. We performed this study to disclose different miRNA-mRNA networks of obesity-related ccRCC in VAT and SAT using datasets from Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA); and find out different RNAs correlated with the prognosis of ccRCC in VAT and SAT. METHODS: We screened out different expressed (DE) mRNAs and miRNAs of obesity, in both VAT and SAT from GEO datasets, and constructed miRNA-mRNA networks of obesity-related ccRCC. To evaluate the sensitivity and specificity of RNAs in networks of obesity-related ccRCC in both VAT and SAT, Receiver Operating Characteristic (ROC) analyses were conducted using TCGA datasets. Spearman correlation analyses were then performed to find out RNA pairs with inverse correlations. We also performed Cox regression analyses to estimate the association of all DE RNAs of obesity with the overall survival. RESULTS: 136 and 185 DE mRNAs of obesity in VAT and SAT were found out. Combined with selected DE miRNAs, miRNA-mRNA networks of obesity-related ccRCC were constructed. By performing ROC analyses, RNAs with same trend as shown in networks and statistically significant ORs were selected to be paired. Three pairs were finally remained in Spearman correlation analyses, including hsa-miR-182&ATP2B2, hsa-miR-532&CDH2 in VAT, and hsa-miR-425&TFAP2B in SAT. Multivariable Cox regression analyses showed that several RNAs with statistically significant adjusted HRs remained consistent trends as shown in DE analyses of obesity. Risk score analyses using selected RNAs showed that the overall survival time of patients in the low-risk group was significantly longer than that in the high-risk group regardless of risk score models. CONCLUSIONS: We found out different miRNA-mRNA regulatory networks of obesity-related ccRCC for both VAT and SAT; and several DE RNAs of obesity-related ccRCC were found to remain consistent performance in terms of ccRCC prognosis. Our findings could provide valuable evidence on the targeted therapy of obesity-related ccRCC.'},\n",
       " {'title': 'Epigenetic clock and DNA methylation analysis of porcine models of aging and obesity.',\n",
       "  'author': ['Schachtschneider KM',\n",
       "   'Schook LB',\n",
       "   'Meudt JJ',\n",
       "   'Shanmuganayagam D',\n",
       "   'Zoller JA',\n",
       "   'Haghani A',\n",
       "   'Li CZ',\n",
       "   'Zhang J',\n",
       "   'Yang A',\n",
       "   'Raj K',\n",
       "   'Horvath S'],\n",
       "  'abstract': \"DNA-methylation profiles have been used successfully to develop highly accurate biomarkers of age, epigenetic clocks, for many species. Using a custom methylation array, we generated DNA methylation data from n = 238 porcine tissues including blood, bladder, frontal cortex, kidney, liver, and lung, from domestic pigs (Sus scrofa domesticus) and minipigs (Wisconsin Miniature Swine). Samples used in this study originated from Large White X Landrace crossbred pigs, Large White X Minnesota minipig crossbred pigs, and Wisconsin Miniature Swine. We present 4 epigenetic clocks for pigs that are distinguished by their compatibility with tissue type (pan-tissue and blood clock) and species (pig and human). Two dual-species human-pig pan-tissue clocks accurately measure chronological age and relative age, respectively. We also characterized CpGs that differ between minipigs and domestic pigs. Strikingly, several genes implicated by our epigenetic studies of minipig status overlap with genes (ADCY3, TFAP2B, SKOR1, and GPR61) implicated by genetic studies of body mass index in humans. In addition, CpGs with different levels of methylation between the two pig breeds were identified proximal to genes involved in blood LDL levels and cholesterol synthesis, of particular interest given the minipig's increased susceptibility to cardiovascular disease compared to domestic pigs. Thus, breed-specific differences of domestic and minipigs may potentially help to identify biological mechanisms underlying weight gain and aging-associated diseases. Our porcine clocks are expected to be useful for elucidating the role of epigenetics in aging and obesity, and the testing of anti-aging interventions.\"},\n",
       " {'title': 'Conditional Deletion of AP-2beta in the Periocular Mesenchyme of Mice Alters Corneal Epithelial Cell Fate and Stratification.',\n",
       "  'author': ['Walker H',\n",
       "   'Taiyab A',\n",
       "   'Deschamps P',\n",
       "   'Williams T',\n",
       "   'West-Mays JA'],\n",
       "  'abstract': 'The cornea is an anterior eye structure specialized for vision. The corneal endothelium and stroma are derived from the periocular mesenchyme (POM), which originates from neural crest cells (NCCs), while the stratified corneal epithelium develops from the surface ectoderm. Activating protein-2beta (AP-2beta) is highly expressed in the POM and important for anterior segment development. Using a mouse model in which AP-2beta is conditionally deleted in the NCCs (AP-2beta NCC KO), we investigated resulting corneal epithelial abnormalities. Through PAS and IHC staining, we observed structural and phenotypic changes to the epithelium associated with AP-2beta deletion. In addition to failure of the mutant epithelium to stratify, we also observed that Keratin-12, a marker of the differentiated epithelium, was absent, and Keratin-15, a limbal and conjunctival marker, was expanded across the central epithelium. Transcription factors PAX6 and P63 were not observed to be differentially expressed between WT and mutant. However, growth factor BMP4 was suppressed in the mutant epithelium. Given the non-NCC origin of the epithelium, we hypothesize that the abnormalities in the AP-2beta NCC KO mouse result from changes to regulatory signaling from the POM-derived stroma. Our findings suggest that stromal pathways such as Wnt/beta-Catenin signaling may regulate BMP4 expression, which influences cell fate and stratification.'},\n",
       " {'title': 'Genome-Wide Analysis for Early Growth-Related Traits of the Locally Adapted Egyptian Barki Sheep.',\n",
       "  'author': ['Abousoliman I',\n",
       "   'Reyer H',\n",
       "   'Oster M',\n",
       "   'Murani E',\n",
       "   'Mohamed I',\n",
       "   'Wimmers K'],\n",
       "  'abstract': 'Sheep play a critical role in the agricultural and livestock sector in Egypt. For sheep meat production, growth traits such as birth and weaning weights are very important and determine the supply and income of local farmers. The Barki sheep originates from the northeastern coastal zone of Africa, and due to its good adaptation to the harsh environmental conditions, it contributes significantly to the meat production in these semi-arid regions. This study aimed to use a genome-wide SNP panel to identify genomic regions that are diversified between groups of individuals of Egyptian Barki sheep with high and low growth performance traits. In this context, from a phenotyped population of 140 lambs of Barki sheep, 69 lambs were considered for a genome-wide scan with the Illumina OvineSNP50 V2 BeadChip. The selected lambs were grouped into divergent subsets with significantly different performance for birth weight and weaning weight. After quality control, 63 animals and 40,383 SNPs were used for analysis. The fixation index (F(ST)) for each SNP was calculated between the groups. The results verified genomic regions harboring some previously proposed candidate genes for traits related to body growth, i.e., EYA2, GDF2, GDF10, MEF2B, SLC16A7, TBX15, TFAP2B, and TNNC2. Moreover, novel candidate genes were proposed with known functional implications on growth processes such as CPXM2 and LRIG3. Subsequent association analysis showed significant effects of the considered SNPs on birth and weaning weights. Results highlight the genetic diversity associated with performance traits and thus the potential to improve growth traits in the Barki sheep breed.'},\n",
       " {'title': 'Transcriptomics based multi-dimensional characterization and drug screen in esophageal squamous cell carcinoma.',\n",
       "  'author': ['Li Y', 'Xu F', 'Chen F', 'Chen Y', 'Ge D', 'Zhang S', 'Lu C'],\n",
       "  'abstract': 'BACKGROUND: Esophageal squamous cell carcinoma (ESCC) remains one of the deadly cancer types. Comprehensively dissecting the molecular characterization and the heterogeneity of ESCC paves the way for developing more promising therapeutics. METHODS: Expression profiles of multiple ESCC datasets were integrated. ATAC-seq and RNA-seq were combined to reveal the chromatin accessibility features. A prognosis-related subtype classifier (PrSC) was constructed, and its association with the tumor microenvironment (TME) and immunotherapy was assessed. The key gene signature was validated in clinical samples. Based on the TME heterogeneity of ESCC patients, potential subtype-specific therapeutic agents were screened. FINDINGS: The common differentially expressed genes (cDEGs) in ESCC were identified. Up-regulated genes (HEATR1, TIMELESS, DTL, GINS1, RUVBL1, and ECT2) were found highly important in ESCC cell survival. The expression alterations of PRIM2, HPGD, NELL2, and TFAP2B were associated with chromatin accessibility changes. PrSC was a robust scoring tool that was not only associated with the prognosis of ESCC patients, but also could reflect the TME heterogeneity. TNS1(high) fibroblasts were associated with immune exclusion. TG-101348 and Vinorelbine were identified as potential subtype-specific therapeutic agents. Besides, the application of PrSC into two immunotherapy cohorts indicated its potential value in assessing treatment response to immunotherapy. INTERPRETATION: Our study depicted the multi-dimensional characterization of ESCC, established a robust scoring tool for the prognosis assessment, highlighted the role of TNS1(high) fibroblasts in TME, and identified potential drugs for clinical use. FUNDING: A full list of funding bodies that contributed to this study can be found in the Acknowledgements section.'},\n",
       " {'title': 'Identification of novel candidate genes implicated in odontogenic potential in the developing mouse tooth germ using transcriptome analysis.',\n",
       "  'author': ['Shin YK',\n",
       "   'Cheon S',\n",
       "   'Kim SD',\n",
       "   'Moon JS',\n",
       "   'Kim JY',\n",
       "   'Kim SH',\n",
       "   'Park C',\n",
       "   'Kim MS'],\n",
       "  'abstract': 'BACKGROUND: In tooth bioengineering for replacement therapy of missing teeth, the utilized cells must possess an inductive signal-forming ability to initiate odontogenesis. This ability is called odontogenic potential. In mice, the odontogenic potential signal is known to be translocated from the epithelium to the mesenchyme at the early bud stage in the developing molar tooth germ. However, the identity of the molecular constituents of this process remains unclear. OBJECTIVE: The purpose of this study is to determine the molecular identity of odontogenic potential and to provide a new perspective in the field of tooth development research. METHODS: In this study, whole transcriptome profiles of the mouse molar tooth germ epithelium and mesenchyme were investigated using the RNA sequencing (RNA-seq) technique. The analyzed transcriptomes corresponded to two developmental stages, embryonic day 11.5 (E11.5) and 14.5 (E14.5), which represent the odontogenic potential shifts. RESULTS: We identified differentially expressed genes (DEGs), which were specifically overexpressed in both the E11.5 epithelium and E14.5 mesenchyme, but not expressed in their respective counterparts. Of the 55 DEGs identified, the top three most expressed transcription factor genes (transcription factor AP-2 beta isoform 3 [TFAP2B], developing brain homeobox protein 2 [DBX2], and insulin gene enhancer protein ISL-1 [ISL1]) and three tooth development-related genes (transcription factor HES-5 [HES5], platelet-derived growth factor D precursor [PDGFD], semaphrin-3 A precursor [SEMA3A]) were selected and validated by quantitative RT-PCR. Using immunofluorescence staining, the TFAP2B protein expression was found to be localized only at the E11.5 epithelium and E14.5 mesenchyme. CONCLUSIONS: Thus, our empirical findings in the present study may provide a new perspective into the characterization of the molecules responsible for the odontogenic potential and may have an implication in the cell-based whole tooth regeneration strategy.'},\n",
       " {'title': 'The Long-Term Dietitian and Psychological Support of Obese Patients Who Have Reduced Their Weight Allows Them to Maintain the Effects.',\n",
       "  'author': ['Ilowiecka K', 'Glibowski P', 'Skrzypek M', 'Styk W'],\n",
       "  'abstract': 'The role of post-therapeutic support after weight loss in obesity treatment is not fully understood. Therefore, weight maintenance after a successful weight loss intervention is not very common, especially in obese individuals. This randomized controlled study was conducted to explore the efficacy of following dietary and psychological support in a group of 36 obese individuals. Participants (22 women, 14 men aged 35.58 +/- 9.85 years, BMI 35.04 +/- 3.80 kg/m(2)) who completed a 12-month weight loss phase (balanced energy-restricted diet) were randomly allocated to receive 18-month support (SG) or no additional care (CG). The support phase included some elements of Ten Top Tips (TTT), cognitive behavioral therapy (CBT), motivational interviewing (MI) in combination with nutritional education and assessment of the level of physical activity. The primary outcome was the maintenance of anthropometric parameters at an 18-month follow-up. The secondary outcomes included evaluation of biochemical parameters and single nucleotide polymorphisms (SNPs) in genes connected with obesity. A comparison of SG vs. CG after a 30-month period of the study revealed significant differences in weight changes (-3.83 +/- 6.09 vs. 2.48 +/- 6.24 kg), Body Mass Index (-1.27 +/- 2.02 vs. 0.72 +/- 2.12 kg/m(2)), visceral adipose tissue (-0.58 +/- 0.63 vs. 0.45 +/- 0.74 L), and waist circumference (-4.83 +/- 4.05 vs. 1.83 +/- 5.97 cm). Analysis of SNPs (rs9939609 FTO, rs987237 TFAP2B, and rs894160 PLIN1) provided further insight into the potential modulating effect of certain genotypes on weight loss and maintenance and extended the knowledge of the potential benefits of personalized medicine. Post-therapeutical support in current clinical practice may increase the chances of long-term weight loss maintenance in obesity treatment even in patients with a genetic predisposition to excessive weight.'},\n",
       " {'title': 'Co-expression of transcription factor AP-2beta (TFAP2B) and GATA3 in human mammary epithelial cells with intense, apicobasal immunoreactivity for CK8/18.',\n",
       "  'author': ['Raap M',\n",
       "   'Gierendt L',\n",
       "   'Werlein C',\n",
       "   'Kuehnle E',\n",
       "   'Kreipe HH',\n",
       "   'Christgen M'],\n",
       "  'abstract': 'AP-2beta is a new mammary epithelial differentiation marker and its expression is preferentially retained and enhanced in lobular carcinoma in situ and invasive lobular breast cancer. In normal breast epithelium AP-2beta is expressed in a scattered subpopulation of luminal cells. So far, these cells have not been further characterized. Co-expression of AP-2beta protein and luminal epithelium markers (GATA3, CK8/18), hormone receptors [estrogen receptor (ER), androgen receptor (AR)] and candidate stem cells markers (CK5/14, CD44) were assessed by double-immunofluorescence staining in normal mammary gland epithelium. The subpopulation of AP-2beta-positive mammary epithelial cells showed an almost complete, superimposable co-expression with GATA3 and a peculiar intense, ring-like appearing immunoreactivity for CK8/18. Confocal immunofluorescence microscopy revealed an apicobasal staining for CK8/18 in AP-2beta-positive cells, which was not seen in in AP-2beta-negative cells. Furthermore, AP-2beta-positive displayed a partial co-expression with ER and AR, but lacked expression of candidate stem cell markers CK5/14 and CD44. In summary, AP-2beta is a new luminal mammary epithelial differentiation marker, which is expressed in the GATA3-positive subpopulation of luminal epithelial cells. These AP-2beta-positive/GATA3-positive cells also show a peculiar CK8/18-expression which may indicate a previously unknown functionally specialized mammary epithelial cell population.'}]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "result_TFAP2B=extrair_info(\"TFAP2B\",\"...@gmail.com\" )\n",
    "gravarBD(result_TFAP2B, \"informaçao TFAP2B\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4336ea7f",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
